src/inverseRobot.js

```diff
@@ -7,7 +7,26 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotKeys = Object.keys(robot);
+  const robotValues = Object.values(robot);
+  const newObject = {};
+
+  for (let i = 0; i < robotValues.length; i++) {
+    for (let j = i + 1; j < robotValues.length; j++) {
+      if (robotValues[i] === robotValues[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
The loop is redundant actually. Just use .hasOwnProperty() method inside the for ... in loop